docs/manual/working/scalaGuide/main/json/ScalaJson.md

```diff
@@ -0,0 +1,186 @@
+<!--- Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com> -->
+# JSON basics
+
+Modern web applications often need to parse and generate data in the JSON (JavaScript Object Notation) format. Play supports this via its [JSON library](https://oss.sonatype.org/service/local/repositories/public/archive/com/typesafe/play/play-json_2.12/2.6.0-M1/play-json_2.12-2.6.0-M1-javadoc.jar/!/play/api/libs/json/).
```

Comment:
I used the sonatype URL as a hack so the docs would still work until we moved it here. @richdougherty Shouldn't it be possible to use the `api/scala/play/api/libs/json/` URL as we use elsewhere now? That would always be the latest version.